plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -446,6 +446,7 @@ public ConnectorTableMetadata getTableMetadata(ConnectorSession session, Connect
         List<ColumnMetadata> columns = getColumns(tableHandle.getMetadataEntry()).stream()
                 .map(column -> getColumnMetadata(column, columnComments.get(column.getName()), columnsNullability.getOrDefault(column.getName(), true)))
                 .collect(toImmutableList());
+        List<String> checkConstraints = getCheckConstraints(tableHandle.getMetadataEntry()).values().stream().collect(toImmutableList());
```

Comment:
These check constrains should be assumed to be expressed in Spark SQL, right?
I think the Delta spec conveniently omits this topic, but we can assume so for now.

Reply:
So, do we want to translate SparkSQL expression into Trino expression?

Reply:
I don't think we should take a shortcut here.
fortunately the check constraints are typically very limited (eg no subqueries expected), so instead of pulling a big library with a big pile of challenges, I would prefer to roll out a Limited SparkSQL Language Subset™ - parser, abstract tree and conversion to Trino form. I actually have it on my todo list.